scripts/docker/CC-Devel/bin/fetchcc.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+set -e
+
+function usage() {
+    echo "${0}"
+    echo "    <Directory of CodeCompass source will be placed to>"
+    echo "    <URL where the CodeCompass source will be fetched from>"
+}
+
+if [[ -z "${2}" ]]; then
+    echo "Too few option(s)." >&2
+    usage
+    exit 1
+fi
+
+if [[ ! -z "${3}" ]]; then
+    echo "Too many options." >&2
+    usage
+    exit 2
+fi
+
+MOUNTED_SRC_DIR="${1}"
+COMPASS_URL="${2}"
+git clone "${COMPASS_URL}" "${MOUNTED_SRC_DIR}"
```

Comment:
Is this Git clone only used for the build, or do we expect the developer to do something meaningful with the checked out working directory?

If **not**, to save space and time, one could clone only the latest commit, etc.

Reply:
It is used only for building stable compass versions. Normally developer container mounts compass source from the developer host.